pkg/controller/direct/firestore/firestoredatabase_controller.go

```diff
@@ -107,13 +102,13 @@ type Adapter struct {
 	id                   *krm.FirestoreDatabaseIdentity
 	firestoreAdminClient *apiv1.FirestoreAdminClient
 	desired              *krm.FirestoreDatabase
-	actual               *firestorepb.Database
+	actual               *pb.Database
 }
 
 var _ directbase.Adapter = &Adapter{}
 
 func (a *Adapter) Find(ctx context.Context) (bool, error) {
-	log := klog.FromContext(ctx).WithName(ctrlName)
+	log := klog.FromContext(ctx)
```

Comment:
Question: Is there a reason we are moving away from using WithName()? It actually seems like it would be helpful.

Reply:
It's already in the context in the logger.

If it isn't, we would add it in the base controller, as that knows the GVK, the controller name, etc etc